toolchain/check/cpp/generate_ast.cpp

```diff
@@ -364,19 +411,35 @@ auto GenerateAst(Context& context,
 
   clang::DiagnosticErrorTrap trap(*diags);
 
-  // Create the AST unit.
-  auto ast = clang::ASTUnit::LoadFromCompilerInvocation(
-      invocation, std::make_shared<clang::PCHContainerOperations>(), nullptr,
-      diags, new clang::FileManager(invocation->getFileSystemOpts(), fs));
-
-  // Attach the AST to SemIR. This needs to be done before we can emit any
-  // diagnostics, so their locations can be properly interpreted by our
-  // diagnostics machinery.
-  context.set_cpp_context(std::make_unique<CppContext>(ast.get()));
+  auto clang = std::make_unique<clang::CompilerInstance>(invocation);
```

Comment:
I personally always get confused by "compiler instance" versus "compiler invocation" (classic human reading optimization - they're similar lengths, both start with "in", etc) - so, sort of related, I worry calling this thing "clang" would confuse me even more (compared to calling it "compiler_instance" here and elsewhere)... but not an especially firm suggestion/critique

Reply:
Hm, also calling it `clang` gives the appearance that it's shadowing the `clang::` namespace, which is probably not great :)

Renamed to `clang_instance`.